py/strato/common/log/coloringformatter.py

```diff
@@ -23,6 +27,10 @@ def format(self, record):
             record.startColor = self._GREEN
         elif record.levelno == _WARNING:
             record.startColor = self._YELLOW
+        elif record.levelno == _STEP:
+            record.startColor = self._LIGHT_GREEN
+            record.levelname = "STEP [%d]" % self.STEP_COUNT
+            self.STEP_COUNT += 1
```

Comment:
I think practically this is not a big issue. ColoringFormatter is used only for StreamHandler in logging library. Potentially there could be a race here where few competing threads will log.step simultaniously... but the only consequence would be two STEP prints with same step number.